components/Buttons/src/MDCButton.m

```diff
@@ -262,6 +256,10 @@ - (void)commonMDCButtonInit {
   // Block users from activating multiple buttons simultaneously by default.
   self.exclusiveTouch = YES;
 
+  // Default background colors.
+  [self setBackgroundColor:MDCColorFromRGB(MDCButtonDefaultBackgroundColor)
+                  forState:UIControlStateNormal];
+
```

Comment:
From what I understand this revert will cause the button color to always be set to the default color when commonMDCButtonInit is called. This means the button background color property will never be nil and the UIAppearance proxy color will never be used. We expect clients to be able to use a UIAppearance proxy from another view controller to set colors for all buttons. Setting background color here will prevent the UIAppearance proxy color from being read.